colin/core/result.py

```diff
@@ -97,6 +99,43 @@ def save_json_to_file(self, file):
                   fp=file,
                   indent=4)
 
+    @property
+    def xunit(self):
+        """
+        Get the xunit representation of results
+
+        :return: str
+        """
+
+        top = Element("testsuites")
+
+        testsuite = SubElement(top, "testsuite")
+
+        for r in self.results:
+            testcase = SubElement(testsuite, "testcase", {
+                'name': r.check_name,
+                # Can't use PASSED or FAILED global variables because their values are PASS
+                # and FAIL respectively and xunit wants them suffixed with -ED.
+                'status': "PASSED" if r.ok else "FAILED",
+                'url': r.reference_url
+            })
+            if len(r.logs) > 0:
```

Comment:
It is enough to check `if r.logs` in python
https://app.codacy.com/manual/user-cont/colin/pullRequest?prid=5390981